pkg/chartutil/requirements.go

```diff
@@ -266,3 +268,128 @@ func ProcessRequirementsEnabled(c *chart.Chart, v *chart.Config) error {
 
 	return nil
 }
+
+// pathToMap creates a nested map given a YAML path in dot notation
```

Comment:
- [ ] I'm new to this code base, but I'm accustomed to seeing Godoc comments punctuated as sentences.